worker.go

```diff
@@ -93,15 +124,47 @@ func startWorkers(fileName string, nbWorkers int, env PreparedEnv) error {
 
 				// TODO: make the max restart configurable
 				if _, ok := workersRequestChans.Load(absFileName); ok {
-					workersReadyWG.Add(1)
```

Comment:
I also encountered this bug in the filewatcher branch. My approach was to entirely ignore the waitgroup after workers have started once, aka remove this line and do something like this:
```
func go_frankenphp_worker_ready() {
	if !workersAreReady.Load() {
		workersReadyWG.Done()
	}
}
```

Reply:
Once metrics are merged, I plan to add a 'workers ready' metric, so I want it to be toggleable.

Reply:
In other news, that specific approach has a race condition (I tried it, among many other implementations) because the `workersAreReady` can become true at different times and cause a deadlock.